setup.cfg

```diff
@@ -62,6 +62,8 @@ invenio_assets.webpack =
     invenio_theme_tugraz_theme = invenio_theme_tugraz.webpack:theme
 invenio_config.module =
     invenio_theme_tugraz = invenio_theme_tugraz.config
+invenio_base.finalize_app =
```

Comment:
please add a separate commit for the finalize_app change

Reply:
done, see PR #323